packages/core/src/root_files/localstorage-sync.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Deriv</title>
+        <meta charset='utf-8' />
+        <meta name='referrer' content='origin' />
+        <script>
+            window.addEventListener('message', function(message) {
+                var restricted_origins = [
+                    'https://smarttrader.deriv.app', 
+                    'https://smarttrader-staging.deriv.app'
+                ];
+
+                // TODO: Remove always true condition.
+                if (true || restricted_origins.includes(message.origin)) {
```

Comment:
Please specify when this TODO should be addressed. Perhaps also add the same tag as the tag for `<SmartTraderIFrame />` if the condition is the same.

Reply:
I've updated this to reflect code we'd use on production, no more TODOs. 👍